### ISSUE TITLE
ContrastiveLoss problem

### ISSUE THREAD

**Issue by zhangjiwei-japan** [Sara-Ahmed/SiT#26]
I think in the code “nominator = torch.exp(positives / self.temperature) # 2*bs
         denominator = self.negatives_mask * torch.exp(similarity_matrix / self.temperature),” denominator contains nominator and the formula (k!=i) in the paper is somewhat different，thank you.

**Comment by Sara-Ahmed** [comment]
Sorry for the late response. The loss function is updated in the main file and the Arxiv paper will be updated soon.
